### PR TITLE
get tests passing

### PR DESCRIPTION
**Pull request by ChrisRackauckas**

### CODE REVIEW

File: test/runtests.jl [hunk 1]
```diff
@@ -128,6 +128,6 @@ end
 
     # Simulate
     estimator = ODEProblem(dynamics(Ψ), u0, tspan, [])
-    sol_ = solve(estimator)
-    @test sol[:,:] ≈ sol_[:,:]
+    sol_ = solve(estimator,Tsit5())
+    @test_broken sol[:,:] ≈ sol_[:,:]
```
Comment (ChrisRackauckas) [inline test/runtests.jl]: @AlCap23 notice the broken marker here.

**Comment by ChrisRackauckas** [comment]
For the broken test:

```julia

ECOS 2.0.5 - (C) embotech GmbH, Zurich Switzerland, 2012-15. Web: www.embotech.com/ECOS

It     pcost       dcost      gap   pres   dres    k/t    mu     step   sigma     IR    |   BT
 0  +0.000e+00  -0.000e+00  +2e+02  9e-01  5e-04  1e+00  4e+00    ---    ---    1  2  - |  -  -
 1  +4.161e-04  +3.248e-04  +2e+00  1e-02  6e-06  1e-02  5e-02  0.9890  1e-04   2  2  2 |  0  0
 2  +1.472e-03  +1.515e-03  +2e-01  3e-03  5e-07  1e-03  4e-03  0.9106  2e-03   3  3  3 |  0  0
 3  +5.335e-03  +5.477e-03  +4e-02  9e-04  1e-07  4e-04  1e-03  0.7723  5e-02   5  4  4 |  0  0
 4  +6.082e-03  +6.236e-03  +3e-02  5e-04  9e-08  3e-04  8e-04  0.4056  2e-01   7  6  6 |  0  0
 5  +6.803e-03  +6.874e-03  +9e-03  2e-04  3e-08  1e-04  3e-04  0.7142  6e-02   8  7  7 |  0  0
 6  +7.606e-03  +7.636e-03  +3e-03  4e-05  9e-09  5e-05  7e-05  0.7759  7e-02   1  1  1 |  0  0
 7  +8.233e-03  +8.252e-03  +9e-04  2e-05  3e-09  3e-05  3e-05  0.8479  2e-01   1  1  1 |  0  0
 8  +8.736e-03  +8.739e-03  +8e-05  2e-06  3e-10  3e-06  2e-06  0.9890  7e-02   1  1  1 |  0  0
 9  +8.780e-03  +8.780e-03  +9e-06  2e-07  3e-11  4e-07  3e-07  0.8995  1e-02   5  5  5 |  0  0
10  +8.787e-03  +8.787e-03  +7e-07  2e-08  3e-12  3e-08  2e-08  0.9631  4e-02   4  4  4 |  0  0
11  +8.788e-03  +8.788e-03  +8e-09  2e-09  1e-13  4e-10  2e-10  0.9883  1e-04   3  3  3 |  0  0

OPTIMAL (within feastol=2.1e-09, reltol=9.3e-07, abstol=8.2e-09).
Runtime: 0.004651 seconds.


ECOS 2.0.5 - (C) embotech GmbH, Zurich Switzerland, 2012-15. Web: www.embotech.com/ECOS

It     pcost       dcost      gap   pres   dres    k/t    mu     step   sigma     IR    |   BT
 0  +0.000e+00  -0.000e+00  +2e+02  9e-01  5e-04  1e+00  4e+00    ---    ---    1  2  - |  -  -
 1  +1.231e-02  +1.224e-02  +2e+01  1e-01  7e-05  1e-01  5e-01  0.8738  1e-04   2  2  2 |  0  0
 2  -2.259e-02  -1.163e-02  +1e+01  2e-02  5e-05  1e-01  4e-01  0.3059  8e-02   3  2  2 |  0  0
 3  -9.248e-03  +1.119e-02  +1e+01  9e-03  4e-05  9e-02  3e-01  0.2526  1e-01   3  2  2 |  0  0
 4  +9.736e-03  +9.899e-03  +1e-01  1e-03  4e-07  1e-03  3e-03  0.9889  1e-04   3  2  2 |  0  0
 5  +1.005e-02  +1.006e-02  +9e-03  8e-05  3e-08  7e-05  2e-04  0.9284  3e-04   5  3  4 |  0  0
 6  +1.055e-02  +1.057e-02  +2e-03  2e-05  8e-09  3e-05  7e-05  0.8007  8e-02   1  3  3 |  0  0
 7  +1.064e-02  +1.065e-02  +1e-03  1e-05  5e-09  2e-05  4e-05  0.4895  2e-01   1  1  1 |  0  0
 8  +1.070e-02  +1.070e-02  +4e-04  4e-06  1e-09  8e-06  1e-05  0.7728  9e-02   1  1  1 |  0  0
 9  +1.077e-02  +1.077e-02  +5e-05  6e-07  2e-10  2e-06  2e-06  0.9525  1e-01   1  1  1 |  0  0
10  +1.080e-02  +1.080e-02  +2e-05  4e-07  7e-11  6e-07  6e-07  0.9890  3e-01   1  2  2 |  0  0
11  +1.082e-02  +1.082e-02  +3e-06  3e-07  3e-11  7e-08  8e-08  0.9251  3e-02   2  3  3 |  0  0
12  +1.083e-02  +1.083e-02  +2e-06  2e-07  3e-11  5e-08  4e-08  0.8661  4e-01   3  3  3 |  0  0
13  +1.083e-02  +1.083e-02  +6e-07  2e-07  2e-11  2e-08  2e-08  0.9890  3e-01   3  3  3 |  0  0
14  +1.084e-02  +1.084e-02  +2e-08  2e-07  1e-11  4e-10  5e-10  0.9890  7e-03   3  3  3 |  0  0
15  +1.085e-02  +1.085e-02  +2e-10  2e-07  2e-11  5e-12  6e-12  0.9890  2e-04   4  4  4 |  0  0
16  +1.086e-02  +1.086e-02  +3e-12  2e-07  2e-11  6e-14  9e-14  0.9890  1e-04   5  5  5 |  0  0
17  +1.086e-02  +1.086e-02  +3e-12  2e-07  2e-11  8e-14  7e-14  0.9890  5e-01   5  4  4 |  0  0
Slacks/multipliers leaving the cone, recovering best iterate (13) and stopping.

Close to OPTIMAL (within feastol=2.0e-07, reltol=5.1e-05, abstol=5.5e-07).
Runtime: 0.007261 seconds.

┌ Warning: Problem status Suboptimal; solution may be inaccurate.
└ @ Convex C:\Users\accou\.julia\packages\Convex\Aro4K\src\solution.jl:51
```

File: test/runtests.jl (hunk 1 above)
Comment (AlCap23) [reply test/runtests.jl]: I will have a look.

**Comment by AlCap23** [comment]
STRridge works fine and gives out the needed result.
I will check the Convex implementation with GLPK and remove ECOS  again.

**Pull request merged**